### DISCUSSION TITLE
Add multilingual support to ForwardTTS

### DISCUSSION THREAD

**Discussion by NeonBohdan** [coqui-ai/TTS#1645] [Ideas]
Multilingual for now is only initialized in VITS
https://github.com/coqui-ai/TTS/blob/c44e39d9d6bfeea15c6e600c6167663c0f9196ea/TTS/tts/models/vits.py#L589-L592

But SpeedySpeech has some pros over glowTTS(part of VITS)
- faster (x2-x3)
- smaller (100MiB->25MiB)

But when I want to train multilingual model I need to get back to VITS
Can this feature be extended to SpeedySpeech?

https://github.com/coqui-ai/TTS/blob/c44e39d9d6bfeea15c6e600c6167663c0f9196ea/TTS/tts/models/forward_tts.py#L181-L185

**Comment by erogol** [comment]
We are currently trying a model for multi-speaker datasets. Next, we can step into multilingual but it has not been experimented with yet.
